app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/AppcoinsClearLogo.kt

```diff
@@ -0,0 +1,119 @@
+package com.aptoide.android.aptoidegames.drawables.icons
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Preview
+@Composable
+fun TestAppcoinsClearLogo() {
+  Image(
+    imageVector = getAppcoinsClearLogo(),
+    contentDescription = null,
+    modifier = Modifier.size(240.dp)
+  )
+}
+
+fun getAppcoinsClearLogo(color: Color = Color(0XFFFF6381)): ImageVector =
```

Comment:
Let's not set the default here.
Better set one of the new colours in the place it is used.

Reply:
fixed